packages/nexus-sdk/src/__tests__/utils.spec.ts

```diff
@@ -39,4 +45,24 @@ describe('utils functions', () => {
       expect(removeLeadingSlash('*')).toEqual('*');
     });
   });
+
+  describe('buildHeader()', () => {
+    it('should return a header string', () => {
+      expect(buildHeader('json')).toEqual('application/ld+json');
+      expect(buildHeader('vnd.graph-viz')).toEqual('text/vnd.graphviz');
+      expect(buildHeader('n-triples')).toEqual('application/n-triples');
```

Comment:
Can you add a test with a random string as args, like `buildHeader('lol')` and make sure it defaults to json?